packages/swap/src/quoting/Quoter.ts

```diff
@@ -1,16 +1,16 @@
+import {
+  assetConstants,
+  ChainflipAsset,
+  getInternalAsset,
+  UncheckedAssetAndChain,
+} from '@chainflip/utils/chainflip';
 import { hexEncodeNumber } from '@chainflip/utils/number';
 import assert from 'assert';
 import BigNumber from 'bignumber.js';
 import { randomUUID } from 'crypto';
 import { Subject, Subscription } from 'rxjs';
 import { Server, Socket } from 'socket.io';
-import {
-  InternalAsset,
-  InternalAssetMap,
-  UncheckedAssetAndChain,
-  assetConstants,
-  getInternalAsset,
-} from '@/shared/enums';
+import { InternalAssetMap } from '@/shared/enums';
```

Comment:
after releasing this - https://github.com/chainflip-io/chainflip-product-toolkit/pull/380 - we can use it from there

Reply:
done

Reply:
we can probably do the same for AssetSymbol, ChainMap, InternalAssetMap, AssetOfChain

Reply:
entire file is yeeted

Reply:
there are some leftovers in the `schema.ts` file

Reply:
fixed that, just need to make sure now that the asset constants that we export match the shape of the old ones